build/AzurePipelinesTemplates/ProjectReunion-RunHelixTests-Job.yml

```diff
@@ -1,28 +1,22 @@
 parameters:
-  name: 'RunTestsInHelix'
+  jobName: 'RunTestsInHelix'
   dependsOn: ''
   condition: ''
-  testSuite: 'DevTestSuite'
-  # If a Pipeline runs this template more than once, this parameter should be unique per build flavor to differentiate the
-  # the different test runs:
-  helixType: 'test/devtest'
-  maxParallel: 4
-  rerunPassesRequiredToAvoidFailure: 5
-  taefQuery: ''
-  # if 'useBuildOutputFromBuildId' is set, we will default to using a build from this pipeline:
-  useBuildOutputFromPipeline: $(System.DefinitionId)
-  matrix: 
+  maxParallel: 10
+  matrix:
     Release_x86:
       buildPlatform: 'x86'
       buildConfiguration: 'Release'
-      outputDirectory: BuildOutput\Release\x86\BaseUnitTests\
     Release_x64:
       buildPlatform: 'x64'
       buildConfiguration: 'Release'
-      outputDirectory: BuildOutput\Release\x64\BaseUnitTests\
+  
+  # Both Azure DevOps and Helix don't have support for arm64.
+  conditionForRunTests: and(succeeded(), ne(variables['buildPlatform'], 'arm64'))
+  conditionForProcessTestResults: and(succeededOrFailed(), ne(variables['buildPlatform'], 'arm64'))
```

Comment:
I don't think you need this.

I added this in the other repo because a I wanted to build the tests and run the tests in the same job, but for arm64 I only wanted to build. It was an ugly hack to get it to work in a single Job (which is good when there is latency on the build machine queue).

I don't think that is necessary for this case, since nothing that interesting is being built in this Job - correct?

Reply:
Right, good point.
 For arm64 it would only be building the two projects
"ProjectReunion.Helix.TestCommon" and "ProjectReunion.Helix.Test.NetCore" -- which are pretty much useless as they don't actually "test" anything.

I'll remove this. Thanks!